wiki/Static-Analysis-Checks.md

```diff
@@ -227,3 +228,11 @@ To fix the failures for this check: resolve the TODO items and then close the is
 
 # How to add a new Static analysis check
 // TODO([#3690](https://github.com/oppia/oppia-android/issues/3690)): Complete static checks Wiki
+
+# How to run static checks locally
+
+- Go to failing CI check in your GitHub PR.
+- Scroll failing CI check logs to the top and find the Bazel command that was run for this script.
+- Alternatively, in android studio, go to the `.github` folder and find the [static_checks.yml](https://github.com/oppia/oppia-android/blob/develop/.github/workflows/static_checks.yml) file. Search for the line that corresponds to the name of the job that failed, and you can run the same script on your local terminal.
```

Comment:
android studio --> Android Studio

...job that failed. You can then run the ...

Reply:
Done.